bin/run_rotgui

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# This file is part of ts_rotgui.
+#
+# Developed for the LSST Telescope and Site Subsystem.
```

Comment:
Please use the modern `Vera Rubin Observatory` header here and in all other files.

Reply:
Updated in the commit: [c90119c](https://github.com/lsst-ts/ts_rotgui/pull/1/commits/c90119c88c04df93ef3f0d07d8af7cfa121ec2ec). Thanks!